lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -134,18 +163,5 @@ $notification-item-height: 72px;
 
     #{$mat-menu-content} {
         padding: 0;
-
-        #{$mat-list} {
-            padding: 8px 0 0;
-
-            #{$mat-list-item-unscoped-content} {
-                display: flex;
-            }
-
-            #{$mat-list-item-content} {
```

Comment:
same here - is that used somewhere yet or can be deleted because I can't see usages?

Reply:
this one might be used in hx repo
![Screenshot 2025-02-14 at 15 23 25](https://github.com/user-attachments/assets/f6b8d077-7532-4c1c-a1be-af5438ca1013)